File: Client/Trunk/Assets/LuaScriptsOld/Common/Debug.lua
-- 输出日志--
function log(str)
    Util.Log(debug.traceback(str));
end

-- 打印字符串--
function print(str)
    Util.Log(debug.traceback(str));
end

-- 错误日志--
function error(str)
    Util.LogError(debug.traceback(str));
end

-- 警告日志--
function warn(str)
    Util.LogWarning(debug.traceback(str));
end

File: Client/Trunk/Assets/Script/Common/Debug.cs
//================================================
// auth：xuetao
// date：2018/5/30 15:32:53
//================================================
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using UnityEngine;

class DebugMgr
{
    public static void LogError(string log)
    {
        UnityEngine.Debug.LogError(log);
    }

    public static void Log(string log)
    {
        UnityEngine.Debug.Log(log);
    }

    public static void LogWarning(string log)
    {
        UnityEngine.Debug.LogWarning(log);
    }
}

